### PR TITLE
Release v0.13.3

### PR DESCRIPTION
**Pull request by daniel-de-vera**
## Change description

Changes for for release `v0.13.3`.

### CODE REVIEW

File: signadot/operator/README.md [hunk 1]
```diff
@@ -68,39 +68,41 @@ kubectl delete ns signadot
...
+| `agent.image`                         | Agent image override                                    | `""`           |
+| `agent.imagePullPolicy`               | Agent image pull policy                                 | `IfNotPresent` |
+| `routeServer.image`                   | Route Server image override                             | `""`           |
+| `routeServer.imagePullPolicy`         | Route Server image pull policy                          | `IfNotPresent` |
+| `ioContextServer.image`               | IO Context Server image override                        | `""`           |
+| `ioContextServer.imagePullPolicy`     | IO Context Server image pull policy                     | `IfNotPresent` |
+| `kubeRBACProxy.image`                 | Kube-rbac-proxy image override                          | `""`           |
+| `kubeRBACProxy.imagePullPolicy`       | Kube-rbac-proxy image pull policy                       | `IfNotPresent` |
+| `routeInit.image`                     | Route Init container image override                     | `""`           |
+| `routeInit.legacy.image`              | Route Init container image override (legacy version)    |                |
```
Comment (daniel-de-vera) [inline signadot/operator/README.md:84]: This one is new.

File: signadot/operator/README.md [hunk 2]
```diff
@@ -68,39 +68,41 @@ kubectl delete ns signadot
...
+| `routeInit.imagePullPolicy`           | Route Init container image pull policy                  | `IfNotPresent` |
+| `routeInit.imagePullSecret`           | Route Init container image pull secret                  | `""`           |
+| `routeSidecar.image`                  | Route Sidecar container image override                  | `""`           |
+| `routeSidecar.legacy.image`           | Route Sidecar container image override (legacy version) | `""`           |
```
Comment (daniel-de-vera) [inline signadot/operator/README.md:88]: This one is new.

**Review by foxish — APPROVED** [review]
LGTM

**Review by foxish — CHANGES_REQUESTED** [review]
Holding this for discussion

**Review by foxish — APPROVED** [review]
LGTM - ok with adding a detailed relnote on the changed meaning of the field.

**Pull request merged**